cucumber/api/features/check_permission.feature

```diff
@@ -12,26 +12,26 @@ Feature: Check whether a role has a privilege on a resource
   Scenario: I confirm that I can perform the granted action
 
     If a role is granted a privilege on a resource, then a permission check will pass.
```

Comment:
what's this? is this the description?  if so, please move it a line upwards and add an empty line before he next line

Reply:
Fixed